wagtail/wagtailcore/tests/test_blocks.py

```diff
@@ -1271,6 +1301,71 @@ class ValidatedBlock(blocks.StreamBlock):
             3: ['Enter a valid URL.'],
         })
 
+    def test_block_level_validation_no_errors(self):
```

Comment:
If I'am not mistaken `render_form` will not be called in `test_block_level_validation_no_errors` and in `test_block_level_validation_throws_errors`.

What do you want to achieve by writing these two tests? Seems like you want to make sure that `ValidatedBlock`, `ColumnBlock`, `IntegerBlock` works correctly, but they are used only for tests porpuses. Don't you think that it looks like writing tests on tests? :)